server/formulas/measurement.js

```diff
@@ -0,0 +1,46 @@
+const MASS = {
+  kg: 1,
+  lb: 2.20462
```

Comment:
maybe we should designate how many sig figs we want. 5 sig figs seems a little unnecessary if everything else is only 2 or less

Reply:
Hmm I'd say leave the calculations as precise as possible, and we can always control the formatting when we actually display the values. That way, we're at least not introducing error.